docs-chef-io/content/inspec/cloud_security.md

```diff
@@ -0,0 +1,207 @@
++++
+title = "Cloud Security"
+draft = false
+gh_repo = "inspec"
+
+[menu]
+  [menu.inspec]
+    title = "Cloud Security"
+    identifier = "inspec/reference/config.md Cloud Security"
+    parent = "inspec/reference"
+    weight = 60
```

Comment:
This will move the page right after the platforms.md page:

```suggestion
    identifier = "inspec/Cloud Security"
    parent = "inspec"
    weight = 35
```

Reply:
ok